portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Usage.testcase

```diff
@@ -66,12 +66,37 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8150 | Test Summary: Page views update in real time"
-	@ignore = "true"
 	@priority = "5"
 	test PageViewsUpdateInRealTime {
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud();
+
+		ACUtils.launchAC();
+
+		ACProperties.switchProperty(propertyName = "${propertyName}");
 
-		// TODO LRAC-8150 Usage#PageViewsUpdateInRealTime pending implementation
+		ACNavigation.goToSettings();
+
+		ACSettings.goToUsage();
+
+		var currentPageViews = selenium.getText("ACUsage#USAGE_CURRENT_PAGE_VIEWS");
+
+		ACUtils.launchDXP();
 
+		ACUtils.closeAllSessions();
+
+		ACUtils.launchAC();
+
+		ACNavigation.goToSettings();
+
+		ACSettings.goToUsage();
+
+		var newPageViews = selenium.getText("ACUsage#USAGE_CURRENT_PAGE_VIEWS");
+
+		if ("${currentPageViews}" != "${newPageViews}") {
+			AssertTextNotEquals(
+				locator1 = "ACUsage#USAGE_CURRENT_PAGE_VIEWS",
+				value1 = "${currentPageViews}");
+		}
```

Comment:
Hi @AfonsoHSantos, I think this `if` is flawed, because if the values are equal, the test will pass and the Assert will not be done, this way the test will not show the possible bug in this number update.

Reply:
Hey @ViniLopes20, make sense. I fixed that